scripts/test.ps1

```diff
@@ -119,9 +119,14 @@ param (
     [switch]$CompressOutput = $false,
 
     [Parameter(Mandatory = $false)]
-    [switch]$NoProgress = $false
+    [switch]$NoProgress = $false,
+
+    [Parameter(Mandatory = $false)]
+    [switch]$EnableAppVerifier = $false
 )
 
+#Requires -RunAsAdministrator
```

Comment:
I'd rather remove this. The script only requires admin if  you use the enable appverifier flag, which won't be used by most people, most of the time, so I'd still like them to be able to run the script without.

Reply:
Is there a way to make this conditional on the appverifier flag being set?

Reply:
Also, the script still requires admin even without the appverifier flag because of the WER dump path setting. And that's not gated behind any switch as far as I can tell.

Reply:
No. There's no way to gate this. It's always on (on all platforms). That's why I think it best to run remove, and warn/error out if a particular feature is used that requires admin.